pages/projects/[id]/index.js

```diff
@@ -70,10 +80,17 @@ export default function ProjectDetailPage() {
       <Heading>{title}</Heading>
       <StyledDepartment>{department}</StyledDepartment>
       <StyledSection>
-        <p>{teamLead}</p>
-        <p>Due date: {endDate}</p>
+        <div>
+          <StyledSectionTitle>Team lead:</StyledSectionTitle>
+          <p>{teamLead}</p>
+        </div>
+        <div>
+          <StyledSectionTitle>Due date:</StyledSectionTitle>
+          <p>{endDate}</p>
+        </div>
```

Comment:
Instead of div + titles, maybe a [Description List](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl?retiredLocale=de) is semantically more appropriate ^^